lib/index.d.ts

```diff
@@ -0,0 +1,43 @@
+import {Request} from 'express';
+import * as passport from 'passport';
+
+export interface StrategyOptions {
```

Comment:
This defines the options for the passport-google-oauth-2 strategy and is taken from the documentation. This "interface" defines the structure of an object, an object matching this structure is a valid StartegyOptions.